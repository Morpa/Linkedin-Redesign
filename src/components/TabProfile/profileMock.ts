export default {
  projects: [
    {
      name: 'Zara redesign concept',
      info: 'UX/UI design, 15.07.2020',
      thumb:
        'https://static.collectui.com/shots/4414691/foody-web-landing-page-design-large'
    },
    {
      name: 'Drozd. Brand identity. 2020',
      info: 'Graphic design, 03.04.2020',
      thumb: 'https://static.collectui.com/shots/4405774/mvmt-watches-medium'
    },
    {
      name: 'SCTHON event brand identity',
      info: 'Graphic design, 03.31.2020',
      thumb:
        'https://static.collectui.com/shots/4028202/credit-card-checkout-medium'
    }
  ],
  skills: [
    {
      name: 'Front End',
      quantity: '6',
      thumb: '/img/front.svg'
    },
    {
      name: 'NextJS',
      quantity: '7',
      thumb: '/img/react.svg'
    },
    {
      name: 'CSS',
      quantity: '5',
      thumb: '/img/css.svg'
    }
  ],
  experiences: [
    {
      title: 'Tech Lead Web',
      company: 'Gold Shake',
      duration: 'Currently',
      description:
        'Responsible for the team of developers working on the client',
      thumb:
        'https://images.unsplash.com/photo-1478186014654-5a7e3898daa5?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=668&q=80'
    },
    {
      title: 'Front end developer',
      company: 'E Corporation',
      duration: '2 years',
      description:
        'Responsible for maintaining and adding new features to the Umbrella Company client platform',
      thumb:
        'https://images.unsplash.com/photo-1606225948173-47c1d329c054?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1500&q=80'
    },
    {
      title: 'Trainee in web development',
      company: 'E Corporation',
      duration: '1 year',
      description:
        'Development of web systems for high potential clients in the niche luxury brands',
      thumb:
        'https://images.unsplash.com/photo-1606225948173-47c1d329c054?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1500&q=80'
    }
  ],
  education: [
    {
      thumb:
        'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAJYAAAC3CAMAAADHL1dDAAAAb1BMVEX///8ZOGd9j6lDXYM1UHlSaY2Mm7PU2uInRHC3wc9vgqDi5uzx8/WaqLwbOmjFzdmotMZgdpYhP2wuSnU+WH/u8PQsSHQ5U3z4+fpMZIiTobd2iKTw8vWEla5XbpDM0968xdNmepqqtsfS2OHe4ul4XODzAAATzUlEQVR4nO1d53ajPLdmU0UTIJoAU839X+ORqMLIMXY8X86Pd681CXZseNDuRYyi/Ef/0X/0H/1bChMgkfXXKB6pgqY0dRL8NY4j2eCzn14R079GcqCmmH6NYP8xkJ0sp2iIOR/r9d9i2SnEWVRAP7/INdM0a//vWWnhnoFA5fwKYYQQAKm9P4R0H8Kw0fnSIH9+R1PZD0ercDL+FShLA06acoJlKPc4T/8IFcqH1LsllQyWMqL+b2BpyW0DcoalhPAnJr+FbgcigUWzP7EVJbEn0iP+kyyaGMeGYcTI4D/1v4DVwIH6Eyyd/AUsM3YnShz+88xExUSn76T+P1BPqwsE623qAhCJbJ1hWQ4G89ug3IKxSt+1611YXoUzc/boX3RNqR4PbpDom1N5E9ZdJ6an+JhBson7NVgV4ucasf8ZrIBM3qgDV7mTb1k0j3pL5NIXVpSYr2AVXAtUAZaLjMn2BmC5MYFvrBYtdYB4uUMWxegRdOwo4rECI9ycYCWzzcD7ORp9VsEOLC024QvCRQ3c+35fzKdqgQnG5AZVZE4UOydYfcItbL/bLQuG+SCEhrR19ntUioo74dUIDEXPF+gd2fLJsj4OMICR8TtEaZQqKT6YGhdCLjtvwlK1+TdFwBwlqhX1N/6yBX2oyU18ixrjcpVPYPkQcbEPXKz9AlYATNib8/tvw/LJZO5uWc5+O7HSw2/Ei6mzZUuU+V1YachZz0xqzNTZxaUNBVY+p+CJ3XsLFvV5bO1sX45QR6IQ/hhWoIPht3vgEEKEtFv4z1arzg5hoCO8Mna71YHeil8NMYCRKmX8r2A9hIESYiuSIiJmGl7PUkgex6rJr2DJndfMxGMYaAqv+s0nqkglQiJrZUU58TOK/hmsC7LFHLyLZZFf/Btz+mtYHYyKSu6nE7i/Kuw8g+UQTdNi8hqWyayml+unvD/Ev4npn8BqUew4DgJjIsJzHQPnwqst86m48xx1VD6gWKphX4UVoIRfRYNoIq3gP3Eyv0I5+1Fo/DAhy7KlPZBQPEO6x7hfg2WTYnLexsEiHpg4E2cimpfJjbDIyJL8Ki+TwRpwM8dOxQVYabakSofMn8aO8huSwPKhn1GVcEhs5LCUMUezHkaNQqN2PccvUgzPdQdo3YWWN+94vtFbD4XuCYTK+XdSbW/VHHeazFE8ixypFk+HKfp4sSwnOzoStCQ+U+mP+V8SOi+dzxRTjTMIh5k6l0ycrD5OE0scqyHztqa90ODAxJ3JOncNaBZjRfhAKvsXG/vrmWc+3BdYisoddAczv10/QpCbbyRBJqinimw5CQRBuhED4ZXkEbrjJ6aavCBbC9G8X2Hx5DWNmc3y7CoH0Gq/B+MyrgAkHovGPAsLEhSpzZy5JA9VyJJnXWdYLMFPF1hMhWiB7NLAkPXDJGgdVm3ft6/UpQ1p/K9OotJEq295rELSnPNZAivlEXMfFwhlYPc84DHK3U2qs+i+tq/ummseiXNAoTznX2BRvRAZUOKIjDJYStHTGkOjmg1i6Xl1XJpbaFF67yE8f+9INkhtsAsdl7pxSRMZBbjfcdlYDdknZLBUvcDVrH22TjrZydvXsHx5cZHCQEvMr2quEe+AtcVCejWOFKORMpEJHWw8uhlEYkvTPHkp+RXWpQQIQcW/zfLP5aMsfyhMOxgqhJmWNE9gGZAlG+Nu+Tl6GBP0uudRE0dKRKsXSV0q8oxoaBBuOCt+2pAJ5RkWkxtgQcd23Q669dDyWU6UBj3WL3gj/1yKncFU2+lQPnjrqtPRmuWGOmBusNyyYX4C6QkB1AO9x7wWOFMy+x63Zik75mp4rd71DJZQaLGKveS9kqvxKGqG5VUYa05Z1qpT21TNmfRUmDjdhKzm0TZ/zUwXDYah7X4Fq0SCWN59VB3+6iIougUW1UjtYyBGOTEnn5ZnVGPAhVOGDjbHICPmpu9PVP8irJZLOu2nO7vV5DHidBjXtHGCpeKWGQ/bZNa8sNlVVw1pyyhnbMME40KQ8anO+zEsJW+49eJKNea4sqNTkd3Ws0TlQm3SfDK1t1ADLT6oHk1dxVMdEYhzqar0FFYIJbUAIsWNc86d+mQDUx2pTCEMOsDqXuzsVYvgYs/KJD4n0+mjB09Vcb1RwSziWUZ79CgUAzgWb3D2+wqxiLH88XoXO3zqHsgde1xtDEnZskWCbn4jJdXDdynkKGeYE9F8VfinoQ03u5z4jwUUvvUQbYxIm29Lx+vtPVQwGRFej2EfERlHix8KIWmSXQ1WW6R353eNfMF5y1fmjUuwYdk+M1GOWobxXNM8rBbzVpLTzYjDLD7XAuQ0okJiSMY93olgtazMYlvl5H9iPUlifpBXHVX6Y41bF1DWWVMPrZt6qTUwUxZdXSuqSZvy4WpdaMjEfrmOo2ssqqvtcflbOph9Bkg1j9wVUbZISEIcFoHUEu8uuz60sreXCo3rEEhYrMJtA/URGMMp4A0qtmoHpT9YplGDvrPDcOimhfLhWvM4l2vGDMuKUe2z0/WkVe4aRJ30s15JQDQuztGi1jgrvfUYiksG4g5ydZ5gUT1ZwgUtLnHyXPHHCJr5yp1jNPFDcd+KgDhhawWmjk0aX7EQJZF7qAmWv5U4mGvuf3RlA+GFHa8BPWrwqSQ4qlMbjfR3boPV106xeuKhJljNfmPVMx+1Uos0SgvC9ddTJR7IDbr7DMdkaaz9gpPPqq0TrJxLspWHyl6k/AEXcfxVIuofBSiIWBj0C1jcencIukuwuJKtsSLVf6yHDLj4zWoxWCy+07fXArnWRpukaHvploeoT8klP8sph5X4Mz1ELQusCnp86FUv5OPdSm42y4duO/yJSTV6me1vAcTDR2dYOfZ97J5gsfTCCVbKN7NNd0GXRHH+Zrabi0NVtwacBw/EYYyYJcXFnEci4fbdQizSIiTJMtF5teY5uTdg3QrChMLVRTfKYAUIsbdDdaJm160gi0XDqmvqmbSzhlibJ1AvdaaoQdrpa7t2UCchBRYjC6qv/tfHxSEMkGXWgialWyrrZIuc3F+XIBjVczNfhOWBrsfooC4djjhK2s8lgBew9uR3LzSP23IZF5r9dzyr0hGWvTXCVxoIcfgc8+OdSmHFu/Sbm3uLkhXpY/QtISOnl2AprsrTvlMYJIWFd/Dt9pVuTpA8Xhl7Re0ahsphjaWqVgUT4nI6lXFu78pgeUJPjG5qS1Fk26GDyIWGmbPa4w1WSndYJWb2IWYxso6ns38AS9kN2xRGxNWFwJmiVQpWWJS5nBWWPRWhp7/cHOguw6KirhmbfxygHa/1f/YG9QqLWfANltYIfymMy7CUTMhh9/Doer+z3LyENYlm2gPqlXSGtdz0Aov7uauwCiGCEKKB+OqwoLMFuC0Qw2kIGdhpOEQGaxGRBRYfsrgKqxJSWGP3Nc3VlHq/rQDURjPMlDdRQ3z7BBbtzL7Q9ITdHkyKS7kkCcnts1j4RMm2rFvfro4nMXoblh0RIFrvVE7BDiYbXYLminJWXp0BirfvbLB8PM3dvQnLTkCv50nVkCBQpwpBGsaFJ/QjLg/bxFuIshUPS0Bc0t+C5Tag2bO3uPUQ9Zq7zI8MYAtmeoCL2xD2You9fmVR43c00UbZaqh485GX1sK5QtCCKRQl7auwdoHdF3iuWXO7VRgCrOd2y4dmWWmWQmsjc34tX/Ri4F1OUcwvw9qNir9d0ZwGZWYrz6PWCRYzaMETWOUW64wF1HQNFWwCeg6lYB+uy9YeJ9RbQDm3c6e/sFQCIaZXCM2lZhmsZi3osNhnilutxfWUuHK6Q/Hrx8xDJB9I3k1Hxm7qatyugF3fNOvGNE1/0lMJrBwvy+ExWZ942Syh0rxo4hal+lq5hs+0m8WUmruCd52S9lO89QQWyWYQwRohDov+OvMZNSHm6yUTkc+IqlxiG3Gb2i1iYiG7M9lqTQpCTS7rnFzElr3VCSw7IyIBinYhLN2J6XcB3eGtzollFvmZ3WKyvtRh5vkHVzXtpUK4B80KJe9N27gZPrWGrzKRwxpQ3C1XbshDWN3uFbRWXnl8Sh45txauw7qtss4T4cNM9ET5ZiGeFdOe0SBppl2G1esbb0ZN0pzeh2zenbqOJBX+niRnImdYMWTOkks3GPXLoZB307VoeSM/910eiRJJ1NgAkdC5QJQgofKwHmExNhxgtRzv7csLQNJbkDLxKh1D1n6eY4zePGMpK4x+EZaXxCMfxnuPh3JR/CIsxY1j6/1hvESWJXwTluLmqHt7GE9qfL8KS0kLeHcYj0qnf74Lizmld7cYbMmuJTa6vgxrnWy6Tmshw54rXWb//wMWhWkiIEXzaEihfQuWXQo+8P3pdB/z/hbLD7kP8vC3YA1YnAX8YGi+4zvkeiPkHtuHL8EKcN+RzSh4xgdbnO961hbOjZ3FjVdYebDT0urwy3MlyA0klKvsRAVVlubU6CBxO8t1SjWCeMKXk3iZYazlU6+nb2rSj9WKkXGjXvB0IySZOoQfzRDTiN/O0Nfp0mqme7NpwP3AqceSwEbrLQl5SwPIIk1bQv+LRw2YS+pzms2hucYl1zWgulrfUm7rbvnuQ/ad6ASrmuYOWR7SXS67KfU2yux139lk+ghrGkv2nKnOcBWWR66NO3wOK42ZeW7z2Z1fhRXi720PlsOKkKuUOJnj16uwjDfS549ghTBwWecy7zFVvAYrxVd6YO9RJKYRLuqZrPN2o99gyItzK1AGa52W7L7GSmpiLFgZI+aybpnMaBalJWvwy2AtA8cWkPK9hPUZBQl2hFSghJhUag648c8t6+ewlpmknt3TUqb6Fd0c0FrFQWs4eOdVNyD9Nod0EVYzGdA7syytdn1m6xkN8bTo7joESAlvaHWS5y+8gFVM7xXToHeYkXe2YJzIjaCZ095yKbLQOHoosVyEZfD7WkdbbhXon2+58wlatZpq+RPfepWJzCGOe6x1L9YbfpesAvo9u7TIk5jtIiy+/Uh4mMTEyfp9TtIaH6cpwydDrRdhlWitO6zkqfhtTgY5fhz1UuUdyYuwAnBOsw6WAc07sUTKrMK5YhNhGa6LsKj06SdDjK8/emrIpKaYGjJcV638vve/dfpt3RgnY+n2nRONDTS7ubur1bZuNJLswrwGq9qHpwY+UWVsysQ4abzmJC1JJsDv+NT99po6j3Mr12ClzT7uGWAomDtN9jsfdKw+jkY/0F07DkjlELPQZbcw5cOUzyVYgY427o+Ix/E+hlgYK6ox6n6EBfFxlBSYXa7EidsgfhipeQlr7tytpALwTao2BnHw+5n1WamIj8uJ+JwBiA2QtIfDdOErWF2OD2azYvFQyp9rIzz/gkq2JB3IFQbzw5AXl2vmg47rY+vg7Hf6M6y2geLB1JizYIWwNzhL/Eroy73bkbCVLYHdh/sg5rRE2FlP9BOsoAH9rP7hJFh32LbRjK+TIsEnA9xZrLu1IUVsqbmNfj+FlfoJ5KHM7dkEUNDDyjiq6a9NqrW5+ZiJ+7A/usdpRB30fI0Hg+kTWFOc39giqHHY+BAgwIBX5pr4SjvKX32fyQI+tO0jSNktHhlyr3XAmkYedoimIcr5xtbyYElufGPxFtdYMWweMsDXHsIQLTsIacSHTzdhHAsQI535PT/i+xww0pPCMAwtiQm34cQJH6zbPQeiC9sUx22Ea6ouXaFUzxcrMfSqcPqST4B1p4+3ZpRkExrABMVa75/jO2YOmBmOYE/stp2ExeWNKxaRZcAuszQJfPYwDk8HFsUHkilv58c9QEeyQdJBbgCltP4wZwl42a2HU2nZhHeawuU5OLLh0sjqM2JOhxcJo6NwhvBeGUd9vAsvFm+11qQG6ZFSs2kWtaYJQN4AIPHebPxu+c15wFWBMBrKctgXvnX+2PQ4gmH7jqOEaPaIK6rm7SSjP+BqQWArv/Mro4Ya5L4B6zSFyXfEpdHuYG18fb/+To6wIAyIsPG5Bl1fXeeoPlTAqWX7VTcd8QFeF6/LRTXg8xDCWn2CivNtk8cxFjb/tBg6svpZpgnTowC1aF7Nbip2z8eEL2mzBTAjEXfx+hB9mPDX+zc9YdwmYXHNZoF8mLqC0fKQWMViB30930PDrRzzqusC+UKXUgXn4zJEiCVb72qIbwEk2yseKaZ4faqZJ4xehxwRRbswbErjRVdU5ikFKH40woyFxFE3a+8Av4IP+mqB0d4h9aaNZBWcQk8rIdeysGc0Jg/Nd8ZCMu2AXV43UDLFL8BZNVWDxinaFbPGpe2xujoQ/b3JmjOxHOxgmRnT3NbZ99Np0IIxgrbtGubPoViPA87REB6f5AnRi/zrCg1EjLO6iF+y2MxWBpTEJpTO+g4TPXNYPacGGQuHDg420N8dfnhCo3GKs7J1qIRCxq6tg9vA9uwcoSx95zs3RZWjqqy68SH5BB2f6t3AwgaXqSRjm8HykQX5UZTSMBSjL5tvR/wWqqlaedwiuV6rZXah5rKE1qdyjvD0eQVWA682sL1LHYsAJatvc/8LxKOwdg0oPNkN6lY4/n4ng/oZnP9rAtfvlLveszXa+i+6dOspA7U/qOKr5JUM2JMQ17O7n75qOQTV/+xR5dRnMfmlAPDwLdsA9G9WarvEwC7hdG98o1VjuBbL/o5GFUGmBpcu1JosP3S+Zqh+JtpVGZDG/1nZxzBCfGX/p/+NQVAXLJQoqvB+9m/UGmr+uBhN/d9imsnr6ibmzwPQGkc15yduqFVU8PfiRrW/4I8/plsQmk6T6IiHOwTpSePUYfBH/23Bf/T/jf4PWwYtL4v1JicAAAAASUVORK5CYII=',
      school: 'Stamford University ',
      course: 'Web Systems',
      year: '2020 - 2021'
    },
    {
      thumb:
        'https://upload.wikimedia.org/wikipedia/commons/thumb/0/0c/MIT_logo.svg/2560px-MIT_logo.svg.png',
      school: 'Massachusetts Institute of Technology',
      course: 'Computer Science',
      year: '2014 - 2019'
    }
  ]
}
