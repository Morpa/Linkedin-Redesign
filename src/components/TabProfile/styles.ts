import styled, { css } from 'styled-components'

export const Wrapper = styled.div``

const Section = styled.section`
  ${({ theme }) => css`
    width: 100%;
    margin-bottom: ${theme.spacings.small};
    padding: ${theme.spacings.medium};
    background: ${theme.colors.white};
    border-radius: ${theme.border.radius};
  `}
`

export const Title = styled.h2`
  ${({ theme }) => css`
    font-size: ${theme.font.sizes.xlarge};
    text-transform: uppercase;
    margin-bottom: ${theme.spacings.xxxsmall};
  `}
`

export const SeeMore = styled.h4`
  ${({ theme }) => css`
    width: fit-content;
    margin-top: 1.2rem;
    padding: ${theme.spacings.small} 0;
    font-size: 1.4rem;
    color: ${theme.colors.blue};
    cursor: pointer;
    text-transform: uppercase;
  `}
`

export const About = styled(Section)``

export const DescriptionAbout = styled.div`
  height: '4.2rem';
  overflow: hidden;
  text-overflow: ellipsis;
  white-space: nowrap;
`

export const Projects = styled(Section)``

export const Cards = styled.div`
  width: 100%;
  display: flex;
  align-items: center;
  justify-content: space-between;
`

export const Card = styled.div`
  ${({ theme }) => css`
    width: 23rem;
    height: 21.6rem;

    p {
      font-size: ${theme.font.sizes.medium};
      line-height: ${theme.spacings.small};
      cursor: pointer;

      &:hover {
        text-decoration: underline;
      }
    }

    > img {
      width: 100%;
    }
  `}
`

export const Skills = styled(Section)``

export const CardsSkills = styled.div`
  display: flex;
  align-items: center;
  justify-content: space-between;
`

export const CardSkills = styled.div`
  ${({ theme }) => css`
    display: flex;
    flex-direction: column;
    justify-content: flex-start;
    width: 18rem;
    height: 7.2rem;
    border: 0.1rem solid ${theme.colors.gray};
    border-radius: ${theme.border.doubleRadius};

    strong {
      width: 100%;
      padding: ${theme.spacings.xxsmall};
      font-size: ${theme.font.sizes.medium};
      text-transform: uppercase;
      display: flex;
      justify-content: space-between;

      > span {
        color: ${theme.colors.blue};
      }
    }
  `}
`

export const Experience = styled(Section)``

const listInfo = css`
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
`

const infoItem = css`
  ${({ theme }) => css`
    display: flex;
    width: 100%;
    align-items: flex-start;
    justify-content: flex-start;
    padding: ${theme.spacings.xxsmall} 0 ${theme.spacings.small} 0;
    margin-bottom: ${theme.spacings.small};
    border-bottom: 0.1rem solid ${theme.colors.gray};

    &:last-child {
      border-bottom: 0;
      margin: 0;
    }
  `}
`

const descriptionItem = css`
  ${({ theme }) => css`
    margin-left: ${theme.spacings.xxsmall};
    display: flex;
    flex-direction: column;

    > strong {
      font-size: ${theme.font.sizes.medium};
      line-height: 1.2rem;
      margin-bottom: ${theme.spacings.xxsmall};
    }

    > span {
      font-size: 1rem;
      line-height: 1.2rem;
      margin-bottom: 1.2rem;

      > strong {
        color: ${theme.colors.blue};
        margin-left: 1.2rem;
      }
    }

    p {
      font-size: ${theme.font.sizes.small};
      line-height: ${theme.spacings.xsmall};
    }
  `}
`

export const List = styled.div`
  ${listInfo}
`

export const Item = styled.div`
  ${infoItem}
`

export const DescriptionItem = styled.div`
  ${descriptionItem}
`

export const Education = styled(Section)`
  ${DescriptionItem} {
    span + strong {
      font-size: ${({ theme }) => theme.font.sizes.small};
    }
  }
`
