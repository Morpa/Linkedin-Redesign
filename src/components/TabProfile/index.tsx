import * as S from './styles'

const TabProfile = () => (
  <S.Wrapper>
    <h1>TabProfile</h1>
  </S.Wrapper>
)

export default TabProfile
