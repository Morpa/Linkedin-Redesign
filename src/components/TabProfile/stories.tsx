import { Story, Meta } from '@storybook/react/types-6-0'
import TabProfile from '.'

export default {
  title: 'TabProfile',
  component: TabProfile
} as Meta

export const Default: Story = () => <TabProfile />
