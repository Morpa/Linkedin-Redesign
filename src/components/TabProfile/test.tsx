import { render, screen } from '@testing-library/react'

import TabProfile from '.'

describe('<TabProfile />', () => {
  it('should render the heading', () => {
    const { container } = render(<TabProfile />)

    expect(
      screen.getByRole('heading', { name: /TabProfile/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
