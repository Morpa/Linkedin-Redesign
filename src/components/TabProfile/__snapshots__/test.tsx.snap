// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabProfile /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    TabProfile
  </h1>
</main>
`
